vortex-duckdb/src/duckdb/copy_function/callback.rs

```diff
@@ -0,0 +1,109 @@
+use std::ffi::CStr;
+use std::os::raw::{c_char, c_ulong, c_void};
+
+use bitvec::macros::internal::funty::Fundamental;
+use itertools::Itertools;
+use vortex::error::VortexExpect;
+
+use crate::cpp;
+use crate::cpp::{
+    duckdb_data_chunk, duckdb_logical_type, duckdb_vx_copy_func_bind_input, duckdb_vx_error,
+};
+use crate::duckdb::{CopyFunction, Data, DataChunk, LogicalType, try_or, try_or_null};
+
+pub(super) unsafe extern "C" fn bind_callback<T: CopyFunction>(
```

Comment:
I guess this is equivalent to `pub(crate)`? I find it's a bit easier to reason about absolute visibility def rather than relative. wdyt?